### DISCUSSION TITLE
AI comment on commits pencil icon missing

### DISCUSSION THREAD

**Discussion by bradyguyc** [community/community#104368] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

I have a paid individual subscription copilot github.   Recently I purchased the microsoft office individual/home copilot.  It was not working with excel and I had tech support from microsoft on the phone helping me fix that.  In the long run I ended up restoring my windows 11 to factory install and basically starting over with all installations.  I did the keep files not a complete wipe and start over.  Copilot for office now works.  And copilot in visual studio is working except that I don't have the pencil icon that creates commit comments for my code.  Which is one of my most favorite features of the copilot product.  I removed and readded my github account in the upper right hand corner.   I checked my github copilot extensions and upgraded to the latest version.  I am using the latest version of vs 2022 preview. 17.9.0 preview 5.0   The pencil icon was showing up before I reset my laptop on this version of vs.   In the output window I have a message that says sucessfully authorized github copilot.   I tried uninstalling and reinstalling the github copilot extensions.  I was able to uninstall github copilot chat.  But I could not uninstall github copilot.  The uninstall menu option was not available.  Any help would be greatly appreciated.   Thanks

**Comment by ayim** [comment]
If you see GitHub Copilot Chat in the View Menu:
![image](https://github.com/community/community/assets/10951869/d6a6d2c1-1d91-4302-8516-c7a7039ac555)

Make sure "Enable commit message suggestions" is enabled in Tools > Options:
![image](https://github.com/community/community/assets/10951869/5bab7958-6552-465e-81df-fb80bdc1b4d0)

If you're still having issues, lmk

--
Aaron Yim
Product Manager for GitHub Copilot in Visual Studio
github: ayim | aayim@microsoft.com

**Comment by bradyguyc** [comment]
thanks, for the comments.  Unfortunately I have both of these already set.  But still no pencil to add comments.

**Reply by ayim** [reply]
Gotcha. Please file a ticket using "Report a Problem" in VS22 and we'll take a look at logs.
![image](https://github.com/community/community/assets/10951869/95cdc511-31b3-4e94-9a66-cdce98a950f6)

**Reply by ayim** [reply, marked as answer]
Created a temporary issue to track this on our end: https://developercommunity.visualstudio.com/t/AI-comment-on-commits-pencil-icon-missin/10579321

We'd like you to create an issue via "Report a Problem" as well so that we can look into your system logs, thanks!
